### PR TITLE
Vps 056/change to uoa theme

### PR DESCRIPTION
**Pull request by John-Moore-UOA**
## Describe the issue

Theme was for UOA medical sciences.  Changed to general UOA theme

## Describe the solution

Changed colours, as documented 

## Risk

The test cases where changed (the image and alt), but they should be double checked to ensure they are correct

## Definition of Done

- [ ] Code peer-reviewed
- [ ] Wiki Documentation is written and up to date
- [ ] Unit tests written and passing
- [ ] Integration tests written and passing
- [ ] Continuous Integration build passing
- [ ] Acceptance criteria met
- [ ] Deployed to production environment

## Reviewed By

Who reviewed your PR - for commit history once merged

### CODE REVIEW

File: package.json
Comment (arnard76) [inline package.json:1]: Hey John, 
You may want to undo the changes to this file. I think you said you accidently committed them. 

1. open this branch locally
2. go to `package.json` file and replace whatever you have in that file with with the [`package.json` on the remote master branch](https://github.com/UoaWDCC/VPS/blob/master/package.json)
3. commit that change locally and push that commit to remote branch
4. the PR should update and show no changes to the `package.json` (unless required for the task 🙂)

File: frontend/public/uoa-logo.png
Comment (arnard76) [inline frontend/public/uoa-logo.png:1]: I guess we can delete this file then?

https://github.com/UoaWDCC/VPS/blob/master/frontend/public/uoa-med-and-health-sci-logo.png

**Comment by arnard76** [comment]
Running locally and it works. Blue UoA theme looks really cool 👌👌

File: package.json
Comment (John-Moore-UOA) [reply package.json:1]: yes, I have now committed the reverted file. Thanks

File: frontend/public/uoa-logo.png
Comment (John-Moore-UOA) [reply frontend/public/uoa-logo.png:1]: Yes, The UoA logo has replaced this file

**Review by djos192 — APPROVED** [review]

**Pull request merged**